src/puppetlabs/puppetdb/scf/migrate.clj

```diff
@@ -1266,6 +1265,11 @@
     "create index reports_job_id_idx on reports(job_id) where job_id is not null"
     "create index catalogs_job_id_idx on catalogs(job_id) where job_id is not null"))
 
+(defn add-latest-report-timestamp-to-certnames []
+  (jdbc/do-commands
+    "ALTER TABLE certnames ADD COLUMN latest_report_timestamp timestamp with time zone"
+    "CREATE INDEX idx_certnames_latest_report_timestamp ON certnames(latest_report_timestamp)"))
```

Comment:
we will want to backfill the column with existing data. I'd guess a simple update would be alright since it's a relatively small table.